build_tools/azure/install.sh

```diff
@@ -71,6 +67,8 @@ if [[ "$DISTRIB" == "conda" ]]; then
         pip install pytest-xdist
     fi
 
+    python -m pip install pandas
```

Comment:
Why this change?

Reply:
I was debugging something, this change will be reverted. (Forgot dictionaries are not ordered in python 3.5)